src/main/java/pl/javastart/task/model/Truck.java

```diff
@@ -0,0 +1,51 @@
+package pl.javastart.task.model;
+
+public class Truck extends Car {
+    private static final double TRUCK_AIR_CONDITIONING_FUEL_CONSUMPTION = 1.6;
+    private static final double LOAD_FUEL_CONSUMPTION = 0.5;
+    private double loadWeight;
+
+    public Truck(String name, double tankCapacity, double avgFuelConsumptionPer100,
+                 boolean airConditioningOn, double loadWeight) {
+        super(name, tankCapacity, avgFuelConsumptionPer100, airConditioningOn);
+        this.loadWeight = loadWeight;
+    }
+
+    public double getLoadWeight() {
+        return loadWeight;
+    }
+
+    public void setLoadWeight(double loadWeight) {
+        this.loadWeight = loadWeight;
+    }
+
+    @Override
+    public double countRange() {
+        return super.countRange();
+    }
```

Comment:
Jeżeli nie zmieniasz logiki z klasy nadrzędnej, to nie ma potrzeby tego zapisywać